near-sdk-macros/src/core_impl/code_generator/impl_item_method_info.rs

```diff
@@ -121,7 +121,9 @@ impl ImplItemMethodInfo {
                     match ::near_sdk::env::input() {
                         Some(input) => match ::near_sdk::serde_json::from_slice(&input) {
```

Comment:
if this is decided to be proceeded with, 
 
`near-sdk-macros` has to be grepped for `near_sdk::serde_json::from_slice` and checked how error is formatted there , as there's at least one more spot, where deserialize input is used - in `#[callback_...]` attributes (where https://docs.rs/near-sdk/5.14.0/near_sdk/env/fn.promise_result.html is used instead of https://docs.rs/near-sdk/5.14.0/near_sdk/env/fn.input.html). 

it appears to me the error should be printed uniformly in all such spots